frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx

```diff
@@ -39,6 +39,8 @@ const ConfiguratePodcastForm: React.FC<Props> = ({
 
   const isFormDisabled = createPodcastStatus === DataStatus.PENDING;
 
+  const sortedGenres = [...genres].sort((a, b) => a.label > b.label ? 1 : -1);
```

Comment:
you need to create a function from it and move it to `frontend/src/components/configurate-podcast/components/configurate-podcast-form/helpers`